### DISCUSSION TITLE
What is the difference between latest and dev?

### DISCUSSION THREAD

**Discussion by stepanov1975** [haugene/docker-transmission-openvpn#2372] [Q&A] [answered]
I see there are no new versions for a long time but seems the latest container is still updated. So is the latest actually the latest or the dev s latest?

**Comment by pkishino** [comment]
latest is built using master branch (which hasn:t been updated in a long time) and is built on a schedule..
dev branch contains the most up to date build as this is where development currently occurs.
Due to time/family/other commitments, Haugene has in practice left the project and I just try to do some housekeeping here and there. The container runs perfectly fine for most users and there isn:t much need to change or update things, except the vpn profiles that do change over time, but they have been extracted to a separate repo

**Reply by jes1417** [reply]
Thank you for keeping us going

**Reply by stepanov1975** [reply]
@pkishino 
So if I want to upgrade from :latest to :dev branch, do I need to do something special or just change the container label to pull to :dev and recreate it?

**Reply by kenske** [reply]
@pkishino Has there been any consideration on forking the project so that new releases are possible?

**Reply by haugene** [reply]
:wave: Sorry about the delay. I'm back :) And looking forward to getting up to date and contribute again.

**Comment by pkishino** [comment, marked as answer]
Just change to :dev

**Comment by haugene** [comment]
I just merged all changes into master and tagged a new 4.2 release.
So right now: dev=latest=4.2

I tagged 4.1 at the commit master had before dev was merged.

dev is now updated with the ubuntu:22.04 base image and has newer versions of both OpenVPN and Transmission. Looks to be working fine. I'll give it over the weekend and then if there are not a wave of bugs reported on the :dev tag I'll merge it to master and tag 4.3.
